docs/src/pages/index.js

```diff
@@ -627,28 +627,53 @@ export default function IndexPage() {
           />
           <div className="container20">
```

Comment:
![image](https://user-images.githubusercontent.com/16670475/235827744-d4c68665-b1b0-4d24-b7dd-3d8e1355be16.png)

The height is uneven because on the right card, it has 3 lines of title while the left has only 1. 

My suggestion is that this `container20` class should use CSS Grid, so that the height will be even. Refer to the guide here: https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
Also, might need some `padding-bottom` because it is placed so close to the footer.

Reply:
okay, updated